### DISCUSSION TITLE
GitHub Copilot not redeemed while i'm a verified student

### DISCUSSION THREAD

**Discussion by EslamMagdy12** [community/community#126979] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

I'm a verified student and can't redeem github copilot
![image](https://github.com/community/community/assets/90795109/bc1bee91-238d-4024-87e3-dcc2fb8e41ee)

**Comment by AngelCruzL** [comment]
I have the same issue

**Comment by dlccyes** [comment]
Go to [Settings - Billing and plans - Plans and usage](https://github.com/settings/billing/summary) INSTEAD OF https://github.com/settings/copilot like the official doc claims.
